lib/src/model.ts

```diff
@@ -115,6 +125,10 @@ export class Model extends EventTarget {
     );
   }
 
+  public registerRecognizer(recognizer: KaldiRecognizer) {
+    this.recognizers.set(recognizer.id, recognizer);
```

Comment:
If you register the recognizer on construction wouldn´t it be necessary to unregister it on KaldiRecognizer.remove()?

Reply:
Good idea, I will implement it. 

Recognizers are actually not running on separate threads, so I'm not sure how many you can actually create whilst keeping it performant enough. (I want to add [pthread](https://emscripten.org/docs/porting/pthreads.html) support but don't want to break compatibility with browsers that don't support it so I need to think about it)